packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -512,6 +512,46 @@ export const instructionTypeDefs = /* GraphQL */ `
         newAuthority: Account
     }
 
+    """
+    Token Extension 2022: TransferFee
+    """
+    type TransferFeeExtension {
+        mint: String
+        transferFeeBasisPoints: Int
+        maximumFee: Int
+        transferFeeConfigAuthority: String
+        withdrawWithheldAuthority: String
+        source: String
+        destination: String
+        tokenAmount: TokenAmount
+        feeAmount: TokenAmount
+        signers: [Address]
+        authority: String
+        multisigWithdrawWithheldAuthority: String
+        feeRecipient: String
+        sourceAccounts: [String]
+        multisigtransferFeeConfigAuthority: String
+        multisigAuthority: String
+    }
```

Comment:
`TransferFeeExtension` actually has its own set of instructions, as each extension does. I think you've globbed together all of them here:

- `TransferCheckedWithFee`
- `WithdrawWithheldTokensFromMint`
- `WithdrawWithheldTokensFromAccounts`
- `HarvestWithheldTokensToMint`
- `SetTransferFee`